### PR TITLE
Accessibility attributes to buttons

### PR DESCRIPTION
**Pull request by wildpow**
PR is for issue #266 titled: Accessibility: 'Buttons do not have an accessible name'

I added aria-label/role attributes to carousel and thumbs buttons to pass Chrome dev tool lighthouse aduit.

### CODE REVIEW

File: src/components/Thumbs.js [hunk 1]
```diff
@@ -304,7 +304,7 @@ class Thumbs extends Component {
         return (
             <div className={klass.CAROUSEL(false)}>
                 <div className={klass.WRAPPER(false)} ref={this.setItemsWrapperRef}>
-                    <button type="button" className={klass.ARROW_PREV(!hasPrev)} onClick={this.slideRight} />
+                    <button type="button" className={klass.ARROW_PREV(!hasPrev)} onClick={this.slideRight} role="presentation" aria-label="mobile navigation"/>
```
Comment (leandrowd) [inline src/components/Thumbs.js]: Should the `aria-label` be something like 'next slide / item' and 'previous slide / item'? 

Also, I suggest to make these strings be props. Any string used to communicate with end users need to be translated and using props is a good way to do so.

Comment (Vadorequest) [reply src/components/Thumbs.js]: @leandrowd Totally agree with your statement as a end-user. Good default such as "previous"/"next" is also a good thing.

I'm not sure if the `role=presentation` is really needed, see https://dequeuniversity.com/rules/axe/2.2/button-name?application=lighthouse

**Comment by samaj94** [comment]
Hi, found this PR through https://github.com/leandrowd/react-responsive-carousel/issues/266 because I've been running into similar issues. It would be great if we could resolve the conflicts and merge this in! If this is solved somewhere else and I'm just missing something 👍

File: src/components/Thumbs.js (hunk 1 above)
Comment (wildpow) [reply src/components/Thumbs.js]: @Vadorequest I'm sorry, I maybe reading this wrong but the 4th bullet point says: `role="presentation" or role="none" (ARIA 1.1) and is not in tab order (tabindex="-1").` 
Isn't that saying that a button needs a role?

P.S Sorry it took me so long to get back to this PR, life and stuff...

Comment (wildpow) [reply src/components/Thumbs.js]: @leandrooriente  Do you think there should be two props like `ariaLabelLeft` and ariaLabelRight` or one object with both in it like:
```
<Carousel ariaLable={['crazy prev slide button', `cool next slide button`]} >
```
or 
```
<Carousel ariaLable={{prev: "super cool prev button", next: "the alright next button"}} >
```
and defaults are  `next slide / item` and `previous slide / item` per your suggestion.

**Comment by wildpow** [comment]
> Hi, found this PR through #266 because I've been running into similar issues using the WAVE accessibility tool. It would be great if we could resolve the conflicts and merge this in! If this is solved somewhere else and I'm just missing something 👍



> Hi, found this PR through #266 because I've been running into similar issues using the WAVE accessibility tool. It would be great if we could resolve the conflicts and merge this in! If this is solved somewhere else and I'm just missing something 👍

@Vadorequest I'm sorry, I maybe reading this wrong but the 4th bullet point says: role="presentation" or role="none" (ARIA 1.1) and is not in tab order (tabindex="-1").
Isn't that saying that a button needs a role?

P.S Sorry it took me so long to get back to this PR, life and stuff...

**Comment by wildpow** [comment]
Also did the command for updating snapshots change?

File: src/components/Thumbs.js (hunk 1 above)
Comment (leandrooriente) [reply src/components/Thumbs.js]: Wrong Leandro dude. 

cc: @leandrowd

**Comment by alopes** [comment]
Any updates on this?

**Comment by guily19** [comment]
I started a PR with the parameters as props here -> https://github.com/leandrowd/react-responsive-carousel/pull/352

**Comment by sezna** [comment]
Looks like this was resolved by #352

**Pull request closed without merging**